docs/learn/universal-profile/metadata/read-profile-data.md

```diff
@@ -32,8 +32,7 @@ The parameters to provide to the erc725 instance are:
 - Univeral Profile address: the address of the Universal Profile you want to retrieve data from.
 - Optional only for retrieving decoded data: RPC provider (web3, ethereum, ethers) or plain RPC url of [mainnet](../../../networks/mainnet/parameters.md) or [testnet](../../../networks/testnet/parameters.md) networks.
 
-<details>
-    <summary>Show code for creating an erc725 instance to read data from a Universal Profile</summary>
+The code for creating an erc725 instance to read data from a Universal Profile:
```

Comment:
How about making it for each code snippet like this below?
This will shorten even more the text, as a single sentence like this might not even be read

```
```js title="create instance of erc725.js"
import { ERC725 } from '@erc725/erc725.js';
import profileSchema from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
const erc725js = new ERC725(
  profileSchema,
  "0x03B2689E4843ca56B2A933e7eC1E1ee6C3e6982e", // Universal Profile address
  "https://rpc.testnet.lukso.network",
  {
    ipfsGateway: 'https://api.universalprofile.cloud/ipfs',
  },
);
```
```

Reply:
I am pretty sure there has to be a / after ipfs i.e. `https://api.universalprofile.cloud/ipfs/` because the code replaces `ipfs://` with this string.

Reply:
> I am pretty sure there has to be a / after ipfs i.e. `https://api.universalprofile.cloud/ipfs/` because the code replaces `ipfs://` with this string.

thanks for catching that Andreas 🫡